Apps/phgitlab/readme.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head></head>
+  <body>
+    <h1>GitLab Phantom App</h1>
+    This app connects a GitLab instance with Splunk>Phantom instance. The GitLab App for Phantom allows playbooks to implement various actions from the <a href="https://docs.gitlab.com/ee/api/api_resources.html">GitLab API</a>.
+
+    File: readme.html
+    Copyright (c) Peter Bertel, 2019
```

Comment:
Can change date to 2020 to be consistent with rest of files

Reply:
I updated the date in [this commit](https://github.com/peterbertel/phantom-apps/commit/207fc712790be1ac8fb8b9499bfda6a9d7f6f7bc).